src/sss_client/sss_pam_compat.h

```diff
@@ -54,7 +54,9 @@
 #endif
 
 #ifndef HAVE_PAM_MODUTIL_GETLOGIN
-const char * PAM_NONNULL((1)) pam_modutil_getlogin(pam_handle_t *pamh)
+#include <unistd.h>
+
+static const char * PAM_NONNULL((1)) pam_modutil_getlogin(pam_handle_t *pamh)
```

Comment:
Why static?

Reply:
This header gets included in multiple places, which causes a duplicated symbol error during linking.

Reply:
I would suggest also `inline` (thought most compilers will do with anyway).

Reply:
I missed it was a header file. I agree with the `inline` directive and I see it is already done.